LongWarOfTheChosen/Config/XComAI.ini

```diff
@@ -250,8 +250,8 @@ Child[6]=HuntEnemyWithCover)
 	Child[7]=TryShootOrReloadOrOverwatch, \\
 	Child[8]=HuntEnemyWithCover)
 
-+BehaviorRemovals="SpectreStealthSelector"
-+NewBehaviorsBehaviors=(BehaviorName=SpectreStealthSelector, NodeType=Selector, \\
++BehaviorRemovals="SpectreVisibleSelector"
++NewBehaviorsBehaviors=(BehaviorName=SpectreVisibleSelector, NodeType=Selector, \\
```

Comment:
I guess you didn't notice you still had `NewBehaviorBehaviors` 😆 Don't worry, I'll fix it.